### ISSUE TITLE
How to use only the ray method

### ISSUE THREAD

**Issue by cdliang11** [LCAV/pyroomacoustics#240]
ism_order =0  && ray_tracing=true ?

**Comment by fakufaku** [comment]
Yes, I think so. Although `ism_order=0` will still add the direct path component. If you don't even want that you can set `ism_order=-1`.

**Comment by lixinghe1999** [comment]
@fakufaku 
Hello, my result shows that "only ism, max_order=-1" and "only ray tracing" have very different results, even though the room is just a simple shoebox.
Do you have any hints to solve it? I can provide the code if it is necessary.

**Comment by lixinghe1999** [comment]
This is the result of room.plot_rir when max_order = -1
![ray_tracing](https://user-images.githubusercontent.com/60444469/189792770-396df950-67ed-4823-b253-12515d9bc23d.png)

This is the result of room.plot_rir when room.set_ray_tracing() & max_order = -1
![ism](https://user-images.githubusercontent.com/60444469/189792799-b644222d-155d-4eab-b701-c91617a1ca75.png)

It is clearly that they are different.
